### ISSUE TITLE
getting started as an operator 

### ISSUE THREAD

**Issue by avgur770** [TeleportXYZ/TRIP-Guides#31]
### What's missing from the TRIP Guides

i think there should be info on what someone would need to become an operator.

 for example you would need developer knowledge to build a software, do i just need a computer to host the software built by trip, can i just pay to have hosted on aws, or does trip give for free and anyone can host it.

### Expected content

~ hardware recommendations to host the operators software.
~ does it cost to use the software made by the team
~ other tips
  
### Actual content

not much

**Comment by Alejandro1954-ops** [comment]
I don't think this issue is neeeded. You should comment with your suggestions [here](url) :)
